### ISSUE TITLE
Add new ski OSM-tags to the renderer, please

### ISSUE THREAD

**Issue by 5EOTlGWT** [yvecai/opensnowmap.org#37]
Thanx for best skimap service

Please, add new OSM-tags for nordic pistes

piste:abandoned
piste:oneway

https://wiki.openstreetmap.org/wiki/Piste_Maps

Best regards!

**Comment by yvecai** [comment]
piste:abandoned or abandoned=yes are already taken into account in rendering: these pistes are not rendered.
Maybe you found an issue,can you link to the specific piste you have in mind ?

Now that piste:oneway is taken into account by the router at Opensnowmap.org, you're right, having this not rendered  doesn't help to understand why it doesn't route, sometimes. Will do in the next months.

**Comment by 5EOTlGWT** [comment]
Thank you for taking my point about piste:oneway
I am sure it will make the site even better

Regarding piste:abandoned....
The site specializes in displaying ski data exclusively. And with that in mind, I think it's wrong not to display this tag. It is not superfluous here. Naturally, it should be made very rare dotted and added to the legend.
This will keep these trails from getting lost and maintain a natural interest in them.

Thank you for doing this project! It is very good!

**Comment by yvecai** [comment]
Normally, there's still a faint white line over trees in that case.
Nevertheless, I think Opensnowmap as a showcase, so I doubt it's useful to show something that is not anymore.
In the coming years we'll see, I'm afraid, quite a few pistes disappear. Let's concentrate on maintaining the data we have up to date.

**Comment by 5EOTlGWT** [comment]
Hi yvecai!

I've a comments

1. 
>>> Now that piste:oneway is taken into account by the router at Opensnowmap.org, you're right,
>>> having this not rendered doesn't help to understand why it doesn't route, sometimes. Will do in the next months.
Isn't working. Didn't appear.

2.
Tag piste:name didn't appear.
(not to be confused with the "name" tag)

**Comment by yvecai** [comment]
Hi, agreed, direction arrows haven't be added (yet).
Where do you see the piste:name tag not rendered ? Can you send a link to the osm element or map ?
Normalély, it'shandled:
https://github.com/yvecai/mapnik-opensnowmap.org/blob/master/pistes-carto/project.yml#L1084

**Comment by yvecai** [comment]
**piste:oneway** is now rendered. https://github.com/yvecai/mapnik-opensnowmap.org/commit/e3197aa029392853d89cc74474eb7cda41b51bcb

**Comment by 5EOTlGWT** [comment]
> 



> **piste:oneway** is now rendered. [yvecai/mapnik-opensnowmap.org@e3197aa](https://github.com/yvecai/mapnik-opensnowmap.org/commit/e3197aa029392853d89cc74474eb7cda41b51bcb)

Thanx a lot !

**Comment by 5EOTlGWT** [comment]
> Where do you see the piste:name tag not rendered ? Can you send a link to the OSM element or map ?
> Normally, it's handled:
> https://github.com/yvecai/mapnik-opensnowmap.org/blob/master/pistes-carto/project.yml#L1084

https://www.opensnowmap.org/?zoom=15&lat=57.96477&lon=56.3391&layers=snowbase&marker=false#map=15/56.346/57.967&b=snowmap&m=false&h=true
for compare
- long direct DOTTED section with tag "name" is displayed
- long direct BLACK section with tag "piste:name" isn't displayed

**Comment by 5EOTlGWT** [comment]
OSM element
https://www.openstreetmap.org/way/854991283#map=16/57.9653/56.3453

**Comment by yvecai** [comment]
Yes, you're right: when a piste is a member of a route relation, I expect the piste name to be mapped on the relation, this is why I currently ignore "Телеграф", but also "Вертолет" and so on ways that are member of  this relation https://www.openstreetmap.org/relation/13983710
I'm not sure how / why I should fix this: this seems to be the name of the (summer) tracks, not the black piste name, isn't it ?

However this area make me see something else, I do render way names for piste:abandonned=yes, and probably shouldn't.
